app/model/GuardianWeeklyZones.scala

```diff
@@ -1,26 +1,28 @@
 package model
 
-import com.gu.i18n.CountryGroup
+import com.gu.i18n.{Country, CountryGroup}
 import com.gu.memsub.subsv2.{CatalogPlan, WeeklyPlans}
 
 object GuardianWeeklyZones {
   private val allCountries = CountryGroup.allGroups.flatMap(_.countries).toSet
 
   val zoneACountryGroups = Set(CountryGroup.UK, CountryGroup.US)
 
-  val domesticZoneCountryGroups = Set(
+  val domesticZoneCountryGroups = Seq(
```

Comment:
Are each of the values here unique? I.e no duplicates and expected to never be no duplicates? If so, this should remain a `Set` which has these properties.

Reply:
It's because there needs to be a deliberate order somewhere in the code. Yes, perhaps having that here is needlessly opaque -- any ideas where it would be good for the ordering to live more explicitly?

Reply:
I understand. This is fine then.